soc/arm/nxp_imx/mcimx7_m4/Kconfig.soc

```diff
@@ -25,10 +25,14 @@ config SOC_PART_NUMBER_MCIMX7D5EVM10SC
 config SOC_PART_NUMBER_MCIMX7S3DVK08SA
 	bool
 
+config SOC_PART_NUMBER_MCIMX7D7DVM10SC
```

Comment:
Please move this config up to maintain alpha order.

Reply:
It will be done